packages/shared/src/components/squads/Details.tsx

```diff
@@ -96,29 +107,35 @@ export function SquadDetails({
           id="squad-form"
         >
           {createMode && (
-            <SquadTitle>
-              Let&apos;s set up your <SquadTitleColor>Squad!</SquadTitleColor>
-            </SquadTitle>
+            <>
+              <SquadTitle>Squads early access!</SquadTitle>
+              <SquadSubTitle>
+                Creating a squad allows you to stay up-to-date and communicate
+                privately.
+              </SquadSubTitle>
+            </>
+          )}
+          {!createMode && (
+            <ImageInput
+              initialValue={form.image ?? form.file}
+              id={squadImageId}
+              fallbackImage={cloudinary.squads.imageFallback}
+              className={{
+                container: '!rounded-full border-0 my-1',
+                img: 'object-cover',
+              }}
+              hoverIcon={<CameraIcon size={IconSize.Large} />}
+              alwayShowHover={!createMode && !imageChanged}
+              onChange={() => setImageChanged(true)}
+              size={createMode ? 'medium' : 'large'}
```

Comment:
I think we can now omit the condition.
```suggestion
              size="large"
```